src/sandstorm/sandstorm-http-bridge.c++

```diff
@@ -1129,14 +1130,15 @@ public:
                                       -> kj::Promise<Identity::Client> {
         if (e2.getType() == kj::Exception::Type::DISCONNECTED) {
           // Disconnected. We'll need to reload from disk.
-          Identity::Client newIdentity = loadIdentityFromDisk(textId);
-
-          // Save the new identity to the map so that we don't have to reload it again.
-          auto iter = liveIdentities.find(textId);
-          KJ_ASSERT(iter != liveIdentities.end());
-          iter->second.identity = newIdentity;
+          auto identity = loadIdentityFromDisk(textId);
```

Comment:
By renaming this `newIdentity` -> `identity`, you're now shadowing a captured variable, which is confusing.